src/Numerics/SoaSphericalTensor.h

```diff
@@ -169,8 +210,15 @@ inline SoaSphericalTensor<T>::SoaSphericalTensor(const int l_max, bool addsign)
     }
 }
 
+PRAGMA_OFFLOAD("omp declare target")
 template<typename T>
-inline void SoaSphericalTensor<T>::evaluate_bare(T x, T y, T z, T* restrict Ylm) const
+inline void SoaSphericalTensor<T>::evaluate_bare(const T x,
+                                                 const T y,
+                                                 const T z,
+                                                 T* restrict Ylm,
+                                                 const int Lmax_,
```

Comment:
Do not put underscore. `XXX_` is reserved for class private member variables. Use `lmax` instead.